nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/io/BulkJsonReader.java

```diff
@@ -34,9 +34,9 @@
 import java.time.Instant;
 import java.util.Map;
 
-import org.nuxeo.ecm.core.bulk.BulkParameters;
-import org.nuxeo.ecm.core.bulk.BulkStatus;
-import org.nuxeo.ecm.core.bulk.BulkStatus.State;
+import org.nuxeo.ecm.core.bulk.io.BulkParameters;
+import org.nuxeo.ecm.core.bulk.message.BulkStatus;
+import org.nuxeo.ecm.core.bulk.message.BulkStatus.State;
 import org.nuxeo.ecm.core.io.marshallers.json.EntityJsonReader;
 import org.nuxeo.ecm.core.io.registry.reflect.Setup;
 
```

Comment:
Can we rename this class `BulkStatusJsonReader` ?

Reply:
yes I listed a task to clean the io package but after the merge